gophpserialize.go

```diff
@@ -214,7 +214,13 @@ func (s *Serializer) readValue() interface{} {
 		s.move()
 		return nil
 	}
-	panic("Unknown objType: " + string(objType) + "\n" + string(s.raw))
+
+	panic(fmt.Sprintf(
+		"Unknown object type: `%s` in ...%s... in position: %v",
```

Comment:
Lets follow: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Addressed in https://github.com/zalora/gophpserialize/pull/2/commits/d9e6a478889fe4a793221fe68da75b0498fa5024